archived/DoD/data_processing_utils.py

```diff
@@ -2,7 +2,7 @@
 from collections import defaultdict
 import math
 
-from archived.DoD import FilterType
+# from archived.DoD import FilterType
```

Comment:
if we don't need these we can clean them up directly (as opposed to comment them)